code/src/Components/MovieDetails.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { SINGLE_MOVIE_URL } from 'utils/urls';
+
+// DISPLAY OF SECOND PAGE AND MORE INFO
+
+const MovieDetails = () => {
+  const [movieDetails, setMovieDetails] = useState([]);
+  const {movieTitle} = useParams();
+  const navigate = useNavigate();
+
+  const onBackButtonClick = () => {
+    // navigate(-1);
+    navigate('/')
+  }
+
+  useEffect(()=> {
+    fetch(SINGLE_MOVIE_URL(movieTitle))
+    .then((res) => res.json())
+    .then((data) => {
+      setMovieDetails(data)
+    })
+  }, [movieTitle]);
+
+  return (
+    <div className='more-page'>
```

Comment:
A regular `<div>` could be replaced with a semantic HTML element like `<section>` here